tests/src/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -73,8 +73,9 @@ class WskBasicUsageTests
         Wsk.exists(wsk.usePythonCLI)
     }
 
-    it should "show help and usage info" in {
-        val stdout = wsk.cli(Seq("-h")).stdout
+    it should "show help and usage info using the default language" in {
+        val env = Map("LANG" -> "de_DE")
```

Comment:
When strings are fully translated and de_DE is supported, this test case fail since it explicitly looks for English strings in the output.